frontend/app/src/utils/search.ts

```diff
@@ -19,22 +19,23 @@ export function splitSearch(keyword: Nullable<string>): SplitResult {
     return defaultSplitResult();
 
   const negateOperatorIndex = keyword.indexOf('!=');
-  const equalOperatorIndex = Math.max(
-    keyword.indexOf('='),
-    keyword.indexOf(':'),
-  );
+  const equalOperatorIndex = keyword.indexOf('=');
+  const colonOperatorIndex = keyword.indexOf(':');
+  const matchOperatorIndex = equalOperatorIndex === -1 || colonOperatorIndex === -1
+    ? Math.max(equalOperatorIndex, colonOperatorIndex)
+    : Math.min(equalOperatorIndex, colonOperatorIndex);
```

Comment:
previously, this is not working:
`start=20/12/2023 12:12:12`

The index will point the last `colon` sign, where it should point the `equal` sign.